api/app/src/DataFixtures/AbstractDataFixture.php

```diff
@@ -40,7 +40,7 @@ public function load(ObjectManager $manager)
         }
     }
 
-    public function setContainer(ContainerInterface $container = null)
+    public function setContainer(?ContainerInterface $container = null)
```

Comment:
To be a little nit-picky. If you've added one return type, should at least add it to the other method you've adjusted.

Reply:
Hehe, true but this was adjusted by precommit and not by my fair hand. The others I had to change cos the app was breaking without the changes. I'm sure when we do our code laundry it will bring up all the return types!